src/ui/screens/onboarding/OnboardingStartSlideDesktop.qml

```diff
@@ -15,6 +15,7 @@ ColumnLayout {
     objectName: "onboardingStartSlide"
 
     property string telemetryScreenId: "connect_on_startup"
+    property bool startAtBootCheckbox: true
```

Comment:
A comment might be helpful here. I can see us forgetting why this exists down the road.